edb/common/ast/codegen.py

```diff
@@ -60,7 +60,7 @@ def visit_indented(
         if indent:
             self.new_lines = 1
             self.char_indentation += 1
-        res = super().visit(node)
+        res = self.visit(node)
```

Comment:
_please_ someone check this. This looks right to me, but it also scares the bejeezus out of me.

Reply:
This seems like it should be right, to me.

But also some tests are failing with syntax errors so maybe we're wrong